analyze-slow-queries.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Analyze Slow Queries
+summary: Learn how to locate and analyze slow queries.
+---
+
+# Analyze Slow Queries
+
+To address the issue of slow queries, you need to take the following two steps:
+
+1. Among many queries, identify which type of queries are slow.
+2. Analyze why this type of queries are slow.
+
+You can easily perform step 1 using the [slow query log](/identify-slow-queries.md) and the [statement summary table](/statement-summary-tables.md) features. TiDB Dashboard integrates the two features and directly displays the results in your browser in a convenient way. Therefore, this document focuses on step 2.
+
+Generally, slow queries have the following major causes:
+
+- Optimizer issues, such as wrong index selected, wrong join type or sequence selected.
+- System issues. All issues not caused by the optimizer are system issues. For example, a busy TiKV instance causes requests to be processed slowly; outdated Region information causes slow queries.
+
+In actual situations, optimizer issues might cause system issues. For example, in a certain type of queries, if the optimizer that should use index actually uses full table scan, the SQL queries will consume many resources, causing the CPU usage of some KV instances to soar. This might appear to be a system issue, but in essence it is an optimizer issue.
```

Comment:
```suggestion
In actual situations, optimizer issues might cause system issues. For example, for a certain type of queries, the optimizer uses a full table scan instead of the index. As a result, the SQL queries consume many resources, which causes the CPU usage of some TiKV instances to soar. This seems like a system issue, but in essence, it is an optimizer issue.
```